addOns/webuipoc/src/main/pocs/web-ui/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "web-ui",
```

Comment:
Can you please use camelCase for the name (the directory too), and use something more unique? :)

Reply:
How about reactWebUI?